perl/.yath.rc

```diff
@@ -1,2 +1,3 @@
 [test]
 -I=rel(lib/Nix)
+-I=rel(build/lib/Nix)
```

Comment:
Probably shouldn't assume the build directory. We can pass in a `-I` to `yath` instead, leaving this file as is just for local dev.